repo/packages/C/confluent-connect/2/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "image": "dbtucker/cp-kafka-connect"
```

Comment:
I recommend sticking with an explicit version of the Docker image in the resource.json file vs. making it a variable.

There are a couple reasons why:
* DC/OS UI shows all the versions installed. This won't be accurate anymore.
* There is a tool that allows on-premise DC/OS users to create a local Universe mirror. We need to understand if there are implications if package assets become user configurable.

/cc @pyronicide for his thoughts.